### PR TITLE
Improve GRM doc

### PR DESCRIPTION
**Pull request by jbloom22**

### CODE REVIEW

**Comment by jbloom22** [comment]
@cseed @tpoterba: I'm not sure who wrote GRM. Are the file formats documented somewhere so I can add more information to the docs?

**Comment by danking** [comment]
My intuition is that the reference should be directed the other way. PCA should refer to the GRM docs when discussing the GRM. The GRM docs should actually contain the text explaining the GRM.

**Comment by jbloom22** [comment]
@danking back to you.

File: python/hail/dataset.py [hunk 1]
```diff
@@ -1917,10 +1917,30 @@ def globals(self):
...
+
+        .. math::
+
+          M_{ij} = \\frac{C_{ij}-2p_j}{\sqrt{2p_j(1-p_j)m}},
+
+        with :math:`M_{ij} = 0` for :math:`C_{ij}` missing (i.e. mean genotype imputation). This scaling normalizes genotype variances to a common value :math:`1/m` for variants in Hardy-Weinberg equilibrium and is further motivated in the paper cited above. (The resulting amplification of signal from the low end of the allele frequency spectrum will also introduce noise for rare variants; common practice is to filter out variants with minor allele frequency below some cutoff.)  The factor :math:`1/m` gives each sample row approximately unit total variance (assuming linkage equilibrium) so that the diagonal entries of the GRM are approximately 1. Equivalently,
```
Comment (danking) [inline python/hail/dataset.py]: I think this is a broken back reference:
```
further motivated in the paper cited above
```

File: python/hail/dataset.py [hunk 2]
```diff
@@ -1917,10 +1917,30 @@ def globals(self):
...
+        
+        .. math::
+
+          G_{ik} = \\frac{1}{m} \\sum_{j=1}^m \\frac{(G_{ij}-2p_j)(G_{kj}-2p_j)}{2 p_j (1-p_j)}
+        
+        The output formats are consistent with `PLINK formats <https://www.cog-genomics.org/plink2/formats>_` as created by the `make-rel and make-grm commands <https://www.cog-genomics.org/plink2/distance#make_rel>`_ and used by `GCTA <http://cnsgenomics.com/software/gcta/estimate_grm.html>`_.
```
Comment (danking) [inline python/hail/dataset.py]: I'm pretty sure the underscore comes after the back tick.

**Comment by jbloom22** [comment]
@danking back to you

**Pull request merged**